Source/WebCore/css/parser/CSSPropertyParser.cpp

```diff
@@ -178,6 +178,9 @@ void CSSPropertyParser::addProperty(CSSPropertyID property, CSSPropertyID curren
     // execCommand('FontSizeDelta')).
     ASSERT(isExposed(property, &m_context.propertySettings) || setFromShorthand || isInternal(property));
 
+    if (!implicit && value->isImplicitInitialValue())
+        implicit = true;
```

Comment:
It would be cool to stop using this flag at some point.

Reply:
We will!